solr/core/src/java/org/apache/solr/core/SolrXmlConfig.java

```diff
@@ -132,6 +133,14 @@ public static NodeConfig fromConfig(Path solrHome, XmlConfigFile config, boolean
     String nodeName = (String) entries.remove("nodeName");
     if (Strings.isNullOrEmpty(nodeName) && cloudConfig != null) nodeName = cloudConfig.getHost();
 
+    Map<String, String> coreAdminHandlerActions =
+        readNodeListAsNamedList(
+                config, "solr/coreAdminHandlerActions/*[@name]", "<coreAdminHandlerActions>")
+            .asMap()
+            .entrySet()
+            .stream()
+            .collect(Collectors.toMap(item -> item.getKey(), item -> item.getValue().toString()));
+
```

Comment:
observation: conceptually this fits in the `fillSolrSection` method but that has only `entries` available as its argument and hence this is placed here

Reply:
Yes, exactly! Initially I was also trying to make it inside `fillSolrSection` but it doesn't fits quite nice there :(

Reply:
Can we add this as a code comment?